src/containers/app/sagas/initializeApplication.js

```diff
@@ -8,27 +8,73 @@ import {
   put,
   takeEvery,
 } from '@redux-saga/core/effects';
-import { Logger } from 'lattice-utils';
+import {
+  AppApiActions,
+  AppApiSagas,
+} from 'lattice-sagas';
+import { LangUtils, Logger, ValidationUtils } from 'lattice-utils';
 import type { Saga } from '@redux-saga/core';
 import type { SequenceAction } from 'redux-reqseq';
 
-import { INITIALIZE_APPLICATION, initializeApplication } from '../actions';
-
 import { getEntityDataModelTypes } from '../../../core/edm/actions';
 import { getEntityDataModelTypesWorker } from '../../../core/edm/sagas';
+import { ERR_ACTION_VALUE_TYPE } from '../../../utils/Errors';
+import { INITIALIZE_APPLICATION, initializeApplication } from '../actions';
+
+const { getApp, getAppConfigs } = AppApiActions;
+const { getAppWorker, getAppConfigsWorker } = AppApiSagas;
+
+const { isValidUUID } = ValidationUtils;
+const { isDefined } = LangUtils;
 
 const LOG = new Logger('AppSagas');
 
+const APP_NAME = 'access_requests';
+
 function* initializeApplicationWorker(action :SequenceAction) :Saga<*> {
 
+  const workerResponse :Object = {};
   try {
+    const { value: { match, organizationId, root } } = action;
+    if (!isValidUUID(organizationId)) throw ERR_ACTION_VALUE_TYPE;
+    if (typeof root !== 'string') throw ERR_ACTION_VALUE_TYPE;
+    if (!isDefined(match)) throw ERR_ACTION_VALUE_TYPE;
     yield put(initializeApplication.request(action.id));
+
+    /*
+     * 1. load App
+     */
+
     const responses :Object[] = yield all([
       call(getEntityDataModelTypesWorker, getEntityDataModelTypes()),
+      call(getAppWorker, getApp(APP_NAME))
```

Comment:
Is this the "new" way of getting apps things?

Reply:
No it's still the old way. New way requires several more requests.